### PR TITLE
Create chrome extension popup page

### PR DESCRIPTION
**Pull request by keelywan**
## Popup
chrome-extension-1

### CODE REVIEW

**Review by Aradhya-B — APPROVED** [review]

**Review by andres-holguin — APPROVED** [review]

File: chrome-extension/manifest.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  "name": "Starfish",
+  "version": "1.0",
+  "description": "Create and share course notes!!",
+  "permissions": ["activeTab", "declarativeContent", "storage", "identity",
+      "https://www.googleapis.com/*", "tabs", "*://*.appspot.com/*", "*://*.cloudshell.dev/*"],
+  "background": {
+    "scripts": ["background.js"],
+    "persistent": false
+	},
```
Comment (cftorres) [inline chrome-extension/manifest.json]: nit: indent out

File: chrome-extension/manifest.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  "name": "Starfish",
+  "version": "1.0",
+  "description": "Create and share course notes!!",
+  "permissions": ["activeTab", "declarativeContent", "storage", "identity",
+      "https://www.googleapis.com/*", "tabs", "*://*.appspot.com/*", "*://*.cloudshell.dev/*"],
```
Comment (cftorres) [inline chrome-extension/manifest.json]: nit: make consistent how arrays are formatted, like below perhaps, where each item is on a new line

**Review by cftorres — APPROVED** [review]

File: chrome-extension/popup.css [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+
+.bold {
+  font-weight: bold;
+}
+
+nav {
+  align-items: center;
+  background-color: #004aad;
+  color: white;
+  font-size: 1.75rem;
```
Comment (nickKelman) [inline chrome-extension/popup.css]: Most of these properties seem sorted so I assume this one should be as well.

**Review by nickKelman — APPROVED** [review]
Nice work.

File: chrome-extension/popup.css (hunk 3 above)
Comment (keelywan) [reply chrome-extension/popup.css]: Oh I missed this one, thanks I'll change that!

**Pull request merged**